Packs/MicrosoftTeams/Integrations/MicrosoftTeamsManagement/MicrosoftTeamsManagement.yml

```diff
@@ -0,0 +1,1037 @@
+category: Messaging
+commonfields:
+  id: Microsoft Teams Management
+  version: -1
+configuration:
+- defaultvalue: 3307a0ab-612c-47af-b3b5-8208247562db
+  display: Application ID
+  hidden: false
+  name: app_id
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Manage teams and members in Microsoft Teams.
+display: Microsoft Teams Management
+name: Microsoft Teams Management
+script:
+  commands:
+  - deprecated: false
+    description: Run this command to start the authorization process and follow the
+      instructions in the command results.
+    execution: false
+    name: microsoft-teams-auth-start
+  - deprecated: false
+    description: |-
+      Run this command to complete the authorization process.
+      Should be used after running the microsoft-teams-auth-start command.
+    execution: false
+    name: microsoft-teams-auth-complete
+  - deprecated: false
+    description: Run this command if for some reason you need to rerun the authentication
+      process.
+    execution: false
+    name: microsoft-teams-auth-reset
+  - deprecated: false
+    description: Tests connectivity to Microsoft.
+    execution: false
+    name: microsoft-teams-auth-test
+  - arguments:
+    - default: false
+      description: The name of the team.
+      isArray: false
+      name: display_name
+      required: true
+      secret: false
+    - default: false
+      description: Description of the team.
+      isArray: false
+      name: description
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: public
+      description: 'The visibility of the group and team. Possible values are: "public"
+        and "private". Default is "public".'
```

Comment:
please remove all `Default is "*".` where `defaultValue` is specified.
it looks bad in the product (example from current Teams integration):
![image](https://user-images.githubusercontent.com/30797606/106658960-b6c0d780-65a6-11eb-8de4-c751cd81cf40.png)